jck/build.xml

```diff
@@ -68,10 +66,18 @@
 			</then>
 			<!-- jck materials exist, update jck materials if needed-->
 			<else>
+				<echo message="Running git clean at ${JCK_ROOT_USED}..." />
+				<exec executable="git" dir="${JCK_ROOT_USED}" failonerror="true">
+					<arg value="clean" />
+					<arg value="--force" />
```

Comment:
I believe you also want the `-d` option. For JCK13, there is no `natives` folder in the repo. Git clean will therefore ignore it when cleaning unless you add `-d`.
https://git-scm.com/docs/git-clean#Documentation/git-clean.txt--d

Reply:
Updated pr, added `-d`. Thanks.